### ISSUE TITLE
Sorting of items in the Library level view

### ISSUE THREAD

**Issue by samokus** [mhoeher/opentodolist#82]
I'm a new user with OpenToDoList installed on two Linux and one Android platform.  Its a great app, thank you!

I have a question:  how can I sort the to-do items within a given Todolist "as viewed from the overall library". The aim is to be able to get a complete overview by seeing the most urgent Todos at the top of each TodoList window. Sometimes this is as valuable an overview as the schedule view.

The rest of this message is just clarifying what I mean by that, in case that isn't clear. My understanding of the terms I'm using are:

- Library - the first level of granularity within an account. 
- ToDoList - the level in the hierarchy below Library and Todo
- ToDo - the level below TodoList
- Task - the lowest level of the hierarchy

Hence the "library view" is where we see the contents of just one library with windows for ToDoLists and in each of which the top Todos are visible.

My understanding of how the sort tools operate is
1) The sort option within the todolist view (I don;'t know the proper name for it, I mean where you can add new items to a specific todolist) allows the Todos to be sorted within that view. So far so good.
2) However this sorting is not preserved at the Library view.
3) The sort option at the library view for "Due to" which, I think, only sorts the Todolist windows according to the earliest Due date in each list. 

To recap, is it possible to sort, at the library view level where I can see each Todolist and have the windows manually sorted, but to have the visible Todos in each Todolist sorted by Due date (or, even better, by whatever sort order is selected within each Todolist)?

Thanks for your time in reading this!

**Comment by mhoeher** [comment]
Hi @Geo-Puffin,

thanks for raising this request! I just checked and indeed it seems we don't apply any ordering within the todo list items in the Library page.

I'll created an issue over on GitLab (where the app is actually developed) to track development efforts: https://gitlab.com/rpdev/opentodolist/-/issues/613

I think implementing this should be really straightforward, so maybe I'll get this in the next release (which is due end of March).

**Comment by mhoeher** [comment]
Okay, it *was* actually quite easy to implement this :wink:

The change is already merged to the development branch. If you want, download pre-build binaries from the jobs of this pipeline: https://gitlab.com/rpdev/opentodolist/-/pipelines/791939171

Would be interesting to hear if the change is what you expected.

Otherwise, the change will be part of the next release - so you can latest give it a try once the next version is out.

Closing for now - if you find that the change does not meet your expectations, please reopen.
